packages/wdio-cli/src/commands/run.ts

```diff
@@ -194,10 +196,33 @@ export async function handler(argv: RunCommandArguments) {
             }
         })
 
+        const executeKill = () => {
+            removeSignalListeners()
+            if (tsProcess.pid) {
+                // gracefully kill child process
+                kill(tsProcess.pid, 'SIGINT')
```

Comment:
No need to import process implicitly if we don't do it for others either.
```suggestion
                process.kill(tsProcess.pid, 'SIGINT')
```